patches/tModLoader/Terraria/ModLoader/EquipLoader.cs

```diff
@@ -267,10 +268,10 @@ internal static int GetPlayerEquip(Player player, EquipType type)
 	public static int AddEquipTexture(Mod mod, string texture, EquipType type, ModItem item = null, string name = null, EquipTexture equipTexture = null)
 	{
 		if (!mod.loading)
-			throw new Exception("AddEquipTexture can only be called from Mod.Load or Mod.Autoload");
+			throw new Exception(Language.GetTextValue("tModLoader.LoadErrorLoadOnlyMethod", "AddEquipTexture"));
```

Comment:
Thanks for the localization. While we don't normally bother localizing error messages, as we assume developers can read English, it doesn't hurt here. Please apply to the other relevant places (solution search for `can only be called from Mod.Load`)